src/main/resources/l10n/Menu_es.properties

```diff
@@ -56,7 +56,7 @@ Paste=&Pegar
 Preferences=&Preferencias
 Previous_tab=&Pestaña_anterior
 Quit=&Salir
-Recent_files=Bases_de_datos_&recientes
+Recent_databases=
```

Comment:
Only the key should be changed, not the values. We have no clue, whether all the other translations use a translation of "File" or "Database". Even though, some language use "File", I lean towards a more complete translation instead of having an incomplete one. - In the concrete case here "Bases de datos" seems to be the translation of "databases" and has for sure to be kept.

So, please, just change the key and do not remove the translation.